lib/membrane_opus/parser.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Membrane.Opus.Parser do
+  @moduledoc """
+  Parses self-delimiting Opus stream.
+
+  See https://tools.ietf.org/html/rfc6716#appendix-B for details.
+  """
+  use Membrane.Filter
+
+  alias Membrane.{Buffer, Opus, Stream}
+  alias Membrane.Opus.PacketUtils
+
+  def_input_pad :input,
+    demand_unit: :buffers,
+    caps: [{Opus, self_delimiting?: true}, {Stream, content: one_of([Opus, nil])}]
+
+  def_output_pad :output, caps: {Opus, self_delimiting?: false}
+
+  @impl true
+  def handle_init(_opts) do
+    {:ok, %{acc_payload: <<>>, timestamp: 0}}
```

Comment:
I would add some comment or docs what `acc_payload` mean

Reply:
I think it's pretty obvious, maybe we should change the name instead?

Reply:
Yeah, I see now. I think it's ok or maybe `parsed_payload`? But I don't insist.